lte/gateway/python/integ_tests/defs.mk

```diff
@@ -129,18 +128,19 @@ s1aptests/test_attach_ul_udp_data_with_mobilityd_restart.py \
 s1aptests/test_attach_ul_udp_data_with_multiple_service_restart.py \
 s1aptests/test_attach_ul_udp_data_with_pipelined_restart.py \
 s1aptests/test_attach_ul_udp_data_with_sessiond_restart.py \
-s1aptests/test_service_req_ul_udp_data_with_mme_restart.py \
 s1aptests/test_attach_detach_setsessionrules_tcp_data.py \
 s1aptests/test_enable_ipv6_iface.py \
-s1aptests/test_ipv6_non_nat_dp_ul_tcp.py \
 s1aptests/test_disable_ipv6_iface.py
 
+PRECOMMIT_TESTS_NOT_CONTAINER = s1aptests/test_services_are_running.py \
+s1aptests/test_attach_ul_udp_data.py \
```

Comment:
The `test_services_are_running` is still present in the precommit tests.

Reply:
Yes. As far as I get the name, it should always run as first test in a set of precommit tests. I.e. a bit like the run-before-whatever-test-to-setup kind of "test"-scripts.

Reply:
I remember this just being one general check for running services in the test suite.
Ping @nstng for his intentions for this test.

Reply:
It's not a "setup-test". It does not matter when it's executed.

Reply:
De-duplicated.